integration_tests/tests/forseti/controls/scanner-location.rb

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'json'
+require 'securerandom'
+
+db_password = attribute('forseti-cloudsql-password')
+db_user_name = attribute('forseti-cloudsql-user')
+forseti_server_storage_bucket = attribute('forseti-server-storage-bucket')
+model_name = SecureRandom.uuid.gsub!('-', '')[0..10]
+
+control 'scanner-location' do
+  # Arrange
+  # Copy the constraints from GCS
+  describe command("sudo gsutil cp -r gs://#{forseti_server_storage_bucket}/rules /home/ubuntu/forseti-security") do
+    its('exit_status') { should eq 0 }
+  end
+
+  # Restart Config Validator
+  describe command("sudo systemctl restart config-validator") do
+    its('exit_status') { should eq 0 }
+  end
+
+  inventory_create = command("forseti inventory create --import_as #{model_name}")
+  describe inventory_create do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should match /"id": "([0-9]*)"/}
+  end
+  @inventory_id = /"id": "([0-9]*)"/.match(inventory_create.stdout)[1]
+
+  describe command("forseti model use #{model_name}") do
+    its('exit_status') { should eq 0 }
+  end
+
+  # Act
+  describe command("forseti scanner run") do
```

Comment:
It's safer to run forseti model use and forseti scanner run in the same command, otherwise sometimes the scanner command gets run first `forseti model use #{model_name} && forseti scanner run`.